apps/shipments/iot_client.py

```diff
@@ -36,3 +42,94 @@ def update_shadow(self, device_id, shadow):
             raise AWSIoTError("Error in response from AWS IoT")
 
         return iot_shadow['data']
+
+    def get_list_owner_devices(self, owner_id, next_token=None, results=None, **kwargs):
+        LOG.debug(f'Getting devices for: {owner_id} from AWS IoT')
+        log_metric('transmission.info', tags={'method': 'DeviceAWSIoTClient.get_list_owner_devices'})
+
+        active = kwargs['active']
+        in_bbox = kwargs['in_bbox']
+
+        if in_bbox:
+            in_bbox = ','.join([c.strip() for c in in_bbox.split(',')])
+
+        if not next_token:
+            results = []
+
+        params_dict = OrderedDict(ownerId=owner_id,
+                                  maxResults=settings.IOT_DEVICES_MAX_RESULTS,
+                                  in_bbox=in_bbox if in_bbox else '',
+                                  nextToken=next_token if next_token else '',)
+
+        list_devices = self._get(f'devices?{urlencode(params_dict)}')
```

Comment:
I think it makes sense for the urlencode logic to be moved to the _get function in the base iot_client.py, so that this issue can be avoided in future calls to _get. _get should accept a dictionary of query parameters and perform the urlencode.